explainaboard/processors/processor.py

```diff
@@ -46,10 +47,26 @@ def default_analysis_levels(self) -> list[AnalysisLevel]:
         list would have one level for each."""
         ...
 
-    @abc.abstractmethod
     def default_analyses(self) -> list[Analysis]:
-        """Returns the analyses to be performed."""
-        ...
+        """Returns the analyses to be performed.
+
+        By default it performs analysis over
+        all continuous features specified in the analysis levels."""
```

Comment:
This is not a good idea for several reasons:

- No reason to decide this as the default behavior.
- Users couldn't guess this behavior correctly from the responsibility of "processor".
- Changing this behavior affects every subclasses, which are not always managed by us.

If you need to implement some "common features", introduce an additional utility rather than directly implementing it to the base method.

Not this:
```python
class Base:
  def foo(self):
    one()
    two()  # these are not considered as the Base's functionality

class Derived:
  def foo(self):
    super().foo()
    three()    
```

Implement this instead:
```python
class Base(ABC):
  @abstractmethod
  def foo(self):
    pass

def do_common():  # better to be placed outside Base if possible
  one()
  two()

class Derived:
  def foo(self):
    do_common()
    three()
```

Reply:
OK, this makes sense. It'll make the code a slight bit more verbose when using the default behavior, but I think that's a small price to pay for clarity.

Quick question: why is it better to have "do_common" placed outside of Base?

Reply:
> why is it better to have "do_common" placed outside of Base?
This depends on the situation. In this case, I observed that the "common behavior" is filtering a float features to create a continuous analysis, and the behavior itself is almost independent from the processor.